Packs/PaloAltoNetworks_SecurityAdvisories/Integrations/PaloAltoNetworksSecurityAdvisories/PaloAltoNetworksSecurityAdvisories_test.py

```diff
@@ -62,6 +62,14 @@ def test_fetch_indicators_command(patched_get_advisories):
     for _field, field_value in result[0].get("fields").items():
         assert field_value
 
+    # Tests to cover CVE extraction issue - XSUP-47513
```

Comment:
please explain what you are testing and remove the notion to XSUP-47513. this part is covered in the PR "fixes:..."

Reply:
Done